content/en/docs/concepts/storage/persistent-volumes.md

```diff
@@ -9,7 +9,7 @@ title: Persistent Volumes
 feature:
   title: Storage orchestration
   description: >
-    Automatically mount the storage system of your choice, whether from local storage, a public cloud provider such as <a href="https://cloud.google.com/storage/">GCP</a> or <a href="https://aws.amazon.com/products/storage/">AWS</a>, or a network storage system such as NFS, iSCSI, Gluster, Ceph, Cinder, or Flocker.
+    Automatically mount the storage system of your choice, whether from local storage, a public cloud provider such as <a href="https://cloud.google.com/storage/">GCP</a> or <a href="https://aws.amazon.com/products/storage/">AWS</a>, or a network storage system such as NFS, iSCSI, Gluster, Ceph, Cinder.
```

Comment:
We prefer to list in alphabetical order for third party services and products.. Since we're touching this line, please switch AWS and GCP, and also put the second list in alphabetical order.

Reply:
Done. The second list I still put NFS and iSCSI first because essentially those are different than the rest since they are particular storage protocols compared to specific storage systems. The rest I am sorting to alphabetical